pkg/arvo/sys/zuse.hoon

```diff
@@ -2173,6 +2173,74 @@
     ++  oath  @                                         ::  signature
     --  ::  pki
   --  ::  jael
+::                                                      ::::
+::::                      ++lenz                          ::  (1l) scry network
+  ::                                                    ::::
```

Comment:
This core at minimum seems unrelated to the commit message

Reply:
oops, cross-talk from another line of work -- will remove, apologies